.buildkite/serve.rayci.yml

```diff
@@ -24,9 +26,27 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //python/ray/serve/... //python/ray/tests/...  serve 
         --except-tags post_wheel_build,gpu,worker-container,xcommit
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 3
+        --test-env=EXPECTED_PYTHON_VERSION=3.8
```

Comment:
nit: missing --build-name servepydantic2build

Reply:
this is intentional because that's causing failures 😓 will fix this one in separate PR: https://github.com/ray-project/ray/pull/40675